### PR TITLE
APIView UI improvements

### PR DESCRIPTION
**Pull request by praveenkuttappan**
Combined PR for all changes from feature branch

### CODE REVIEW

File: src/dotnet/APIView/APIViewWeb/APIViewWeb.csproj [hunk 1]
```diff
@@ -2,7 +2,7 @@
 
   <PropertyGroup>
     <TargetFramework>netcoreapp3.1</TargetFramework>
-    <LangVersion>latest</LangVersion>
+    <LangVersion>9</LangVersion>
```
Comment (chidozieononiwu) [inline src/dotnet/APIView/APIViewWeb/APIViewWeb.csproj:5]: Should we change this back to latest?

File: src/dotnet/APIView/APIViewWeb/Controllers/AutoReviewController.cs [hunk 2]
```diff
@@ -2,7 +2,7 @@
 // Licensed under the MIT License.
 
 using APIViewWeb.Filters;
-using APIViewWeb.Respositories;
```
Comment (chidozieononiwu) [inline src/dotnet/APIView/APIViewWeb/Controllers/AutoReviewController.cs:5]: @praveenkuttappan  do you have any idea why this was spelt Respositories in the first place?

File: src/dotnet/APIView/APIViewWeb/Models/PackageGroupMdel.cs [hunk 3]
```diff
@@ -0,0 +1,12 @@
+﻿using System.Collections.Generic;
```
Comment (chidozieononiwu) [inline src/dotnet/APIView/APIViewWeb/Models/PackageGroupMdel.cs:1]: Looks like there is a typo in this file name. Should it be `PackageGroupModel.cs`? Should we fix it?

**Comment by kurtzeborn** [comment]
Duplicate PR.

**Pull request closed without merging**